General Updates (#8553)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/manage-external-access.md

If you want people from other organizations to have access to your teams and channels, use guest access instead. For more information about the differences between external access and guest access, see [Compare external and guest access](communicate-with-users-from-other-organizations.md#compare-external-and-guest-access). 

Use external access when:
  
- You have users in external domains who need to chat. For example, Rob@contoso.com and Ann@northwindtraders.com are working on a project together along with some others in the contoso.com and northwindtraders.com domains.

- You want the people in your organization to use Teams to contact people in specific businesses outside of your organization.

- You want anyone else in the world who uses Teams to be able to find and contact you, using your email address. 

## Plan for external access
External access policies include controls for each type of federation at both the tenant- and user-levels. Turning a policy off at the tenant-level turns it off for all users, regardless of their user-level setting. All external access settings are enabled by default.

> Tenant-level settings can be changed via Teams Admin Center or PowerShell (Set-CSTenantFederationConfiguration). User-level settings can be changed via PowerShell (Set-CsExternalAccessPolicy). In later sections we’ve provided instructions on how to use Teams Admin Center and write PowerShell commands.

Reviewer: ```suggestion
Tenant-level settings can be changed via Teams admin center or PowerShell (Set-CSTenantFederationConfiguration). User-level settings can be changed via PowerShell (Set-CsExternalAccessPolicy). In later sections, we’ve provided instructions on how to use Teams admin center and write PowerShell commands.
```

> For reference, below is a description of the PowerShell controls.

Reviewer: ```suggestion
For reference, the following is a description of the PowerShell controls.
```

6. Click **Save**.

> To communicate with another tenant, they must either enable Allow all external domains or add your tenant to their list of allowed domains by following the same steps above.  

Reviewer: ```suggestion
To communicate with another tenant, they must either enable **Allow all external domains** or add your tenant to their list of allowed domains by following the same steps above.  
```

## Manage contact with external Teams users not managed by an organization

> Admins can choose to enable or disable communications with external Teams users that are not managed by an organization (“unmanaged”). If enabled, they can also further control if external users with unmanaged Teams accounts can initiate contact (see image below). If external user initiation is turned off, unmanaged Teams users will not be able to search the full email address to find organization contacts and all communications with unmanaged Teams users must be initiated by organization users.

Reviewer: ```suggestion
Admins can choose to enable or disable communications with external Teams users that are not managed by an organization (“unmanaged”). If enabled, they can also further control if external users with unmanaged Teams accounts can initiate contact (see the following image). If external user initiation is turned off, unmanaged Teams users will not be able to search the full email address to find organization contacts and all communications with unmanaged Teams users must be initiated by organization users.
```

![image](https://user-images.githubusercontent.com/98335128/155242115-cdd65afe-62c4-40e5-8603-c1ac770103c0.png)

**Using the Microsoft Teams admin center**

> 1.	In the left navigation, go to Users > External access.<br>

Reviewer: ```suggestion
1.	In the left navigation, go to **Users** > **External access**.<br>
```

2.	Follow one of these steps:<br>
      *	To block Teams users in your organization from communicating with external Teams users whose accounts are not managed by an organization  :<br>
               1.	Turn off the People in my organization can communicate with Teams users whose accounts aren't managed by an organization setting <br>
               2.	Clear the External users with Teams accounts not managed by an organization can contact users in my organization checkbox<br>
      *	To let Teams users in your organization communicate with external Teams users whose accounts are not managed by an organization if your Teams users have initiated the contact:<br>
               1.	Turn on the People in my organization can communicate with Teams users whose accounts aren't managed by an organization setting <br>
               2.	Clear the External users with Teams accounts not managed by an organization can contact users in my organization checkbox<br>
      *	To let Teams users in your organization communicate with external Teams users whose accounts are not managed by an organization and receive requests to communicate with those external Teams users: <br>

>                1.	Turn on the People in my organization can communicate with Teams users whose accounts aren't managed by an organization setting<br>
>                2.	Select the External users with Teams accounts not managed by an organization can contact users in my organization checkbox<br>

Reviewer: ```suggestion
               1.	Turn on the _People in my organization can communicate with Teams users whose accounts aren't managed by an organization_ setting.<br>
               2.	Select the _External users with Teams accounts not managed by an organization can contact users in my organization_ checkbox.<br>
```

> 1.	In the left navigation, go to **Users** > **External access**.<br>
> 2.	Follow one of these steps:<br>
>       *	To block Teams users in your organization from communicating with external Teams users whose accounts are not managed by an organization:<br>
>                1.	Turn off the **People in my organization can communicate with Teams users whose accounts aren't managed by an organization** setting. <br>
>                2.	Clear the **External users with Teams accounts not managed by an organization can contact users in my organization** checkbox.<br>
>       *	To let Teams users in your organization communicate with external Teams users whose accounts are not managed by an organization if your Teams users have initiated the contact:<br>
>                1.	Turn on the **People in my organization can communicate with Teams users whose accounts aren't managed by an organization** setting. <br>
>                2.	Clear the **External users with Teams accounts not managed by an organization can contact users in my organization** checkbox.<br>
>       *	To let Teams users in your organization communicate with external Teams users whose accounts are not managed by an organization and receive requests to communicate with those external Teams users: <br>
>                1.	Turn on the **People in my organization can communicate with Teams users whose accounts aren't managed by an organization** setting.<br>
>                2.	Select the **External users with Teams accounts not managed by an organization can contact users in my organization** checkbox.<br>

Reviewer: ```suggestion
1. In the left navigation, go to **Users** > **External access**.<br>
2. Follow one of these steps:<be>
    - To block Teams users in your organization from communicating with external Teams users whose accounts are not managed by an organization:<be>
    	1) Turn off the **People in my organization can communicate with Teams users whose accounts aren't managed by an organization** setting. <br>
        2) Clear the **External users with Teams accounts not managed by an organization can contact users in my organization** checkbox.<br>
    - To let Teams users in your organization communicate with external Teams users whose accounts are not managed by an organization if your Teams users have initiated the contact:<br>
        1) Turn on the **People in my organization can communicate with Teams users whose accounts aren't managed by an organization** setting. <br>
        2) Clear the **External users with Teams accounts not managed by an organization can contact users in my organization** checkbox.<br>
    - To let Teams users In your organization communicate with external Teams users whose accounts are not managed by an organization and receive requests to communicate with those external Teams users: <br>
        1) Turn on the **People in my organization can communicate with Teams users whose accounts aren't managed by an organization** setting.<br>
        2) Select the **External users with Teams accounts not managed by an organization can contact users in my organization** checkbox.<br>
```

## Communicate with Skype users

Follow these steps to let Teams users in your organization chat with and call Skype users. Teams users can then search for and start a one-on-one text-only conversation or an audio/video call with Skype users and vice versa.

![image](https://user-images.githubusercontent.com/98335128/155242238-1e14383e-e7aa-488b-8d73-71d71377090e.png)

  **Using the Microsoft Teams admin center**

1. In the left navigation, go to **Users** > **External access**.

2. Turn on the **Allow users in my organization to communicate with Skype users** setting.

To learn more about the ways that Teams users and Skype users can communicate, including limitations that apply, see [Teams and Skype interoperability](teams-skype-interop.md).

## Limit external access to specific people

> If you've enabled any of the external access controls at a tenant-level, you can limit external access to specific users using PowerShell. People 

Reviewer: @ASolom1, I am not sure about this - should it be "PowerShell. People" (with a space)?

> If you've enabled any of the external access controls at a tenant-level, you can limit external access to specific users using PowerShell. People 

Author: I think that we don't need the word "People" and that was a miss on my part. I updated "people" to "users" earlier in the sentence. Can you review with that context and see if it makes sense?

> You can use the example script below, substituting *Control* for the control you want to change, *PolicyName* for the name you want to give the policy, and *UserName* for each user for which you want to enable/disablebe able to  use external access.

Reviewer: ```suggestion
You can use the following example script, substituting *Control* for the control you want to change, *PolicyName* for the name you want to give the policy, and *UserName* for each user for whom you want to enable/disable external access.
```

|:---------|:-----------------------|
|Online with no Skype for Business on-premises. This includes organizations that have TeamsOnly users and/or Skype for Business Online users.| If using Teams Admin Center:<br>-Make sure People in my organization can communicate with Teams users whose accounts aren’t managed by an organization is enabled in External Access.<br>-If you want unmanaged Teams accounts to initiate chats, check the box for External users with Teams accounts not managed by an organization can contact users in my organization.<br><br>If using PowerShell:<br>-Ensure the tenant is enabled for federation: Get-CsTenantFederationConfiguration must show AllowTeamsConsumer=true.<br>-Ensure the user’s effective value of CsExternalAccessPolicy has EnableTeamsConsumerAccess=true.<br>-Ensure the tenant is enabled for unmanaged users to initiate chats: Get-CsTenantFederationConfiguration must show AllowTeamsConsumerInbound=true.<br>-Ensure the user’s effective value of CsExternalAccessPolicy has EnableTeamsConsumerInbound=true.|
|Pure on-premises | Chat with unmanaged Teams users is not supported for pure on-premises organizations.|

> |Hybrid with some users online (in either Skype for Business or Teams) and some users on-premises. | Follow above steps for online organizations. Please note that chat with unmanaged Teams users is not supported for on-premises users. |

Reviewer: ```suggestion
|Hybrid with some users online (in either Skype for Business or Teams) and some users on-premises. | Follow the previously described steps for online organizations. Note that chat with unmanaged Teams users is not supported for on-premises users. |
```

> [!IMPORTANT]
> You don't have to add any **Skype domains** as allowed domains in order to enable Teams or Skype for Business Online users to communicate with Skype users inside or outside your organization. All **Skype domains** are allowed.

## Test access

To test your setup, you need a Teams user who's not behind your firewall.
  
1. After you and the admin from the organization have changed the **External access** settings, you should be good to go.

2. In the Teams app, search for the person by email address, and send a request to chat.

3. Ask your Teams contact to send you a request to chat. If you don't receive their request, the problem is your firewall settings (assuming they've already confirmed their firewall settings are correct).

> 4. Another way to test whether the problem is your firewall is to go to a WiFi location not behind your firewall. such as a coffee shop, and use Teams to send a request to your contact to chat. If the message goes through at the WiFi location, but does not when you're at work, then you know the problem is your firewall.

Reviewer: ```suggestion
4. Another way to test whether the problem is your firewall is to go to a Wi-Fi location that is not behind your firewall, such as a coffee shop, and use Teams to send a request to your contact to chat. If the message goes through at the Wi-Fi location but does not when you're at work, then you know the problem is your firewall.
```
